rsocket/internal/InMemResumeManager.h

```diff
@@ -91,9 +69,9 @@ class ResumeCache {
   std::shared_ptr<RSocketStats> stats_;
 
   // End position of the send buffer queue
-  ResumePosition position_{0};
+  ResumePosition lastSentPosition_{0};
```

Comment:
Nitpicking, but maybe swap the order of these two so `firstSentPosition_` is the first field?